2.2/alpine3.8/amd64/Dockerfile

```diff
@@ -20,6 +20,7 @@ ENV CORECLR_PROFILER={324F817A-7420-4E6D-B3C1-143FBED6D855}
 ENV CORECLR_PROFILER_PATH_64=/diag/Instrumentation64/libInstrumentationEngine.so
 ENV MicrosoftInstrumentationEngine_FileLog=Errors
 ENV MicrosoftInstrumentationEngine_FileLogPath=/tmp/diag/log.txt
+ENV MicrosoftProductionDiagnostics_RuntimeFolder=/diag/Runtime
```

Comment:
We also need "ENV MicrosoftProductionDiagnostics_UtilsPath=/diag/Utils/ProductionBreakpointsUtils.dll" for each of these.